modules/manage/pages/manage-security/enable-client-certificate-handling.adoc

```diff
@@ -53,7 +53,9 @@ Note that imposing this level of security likely requires the additional measure
 . If *Require Client Certificate* has been set to either *Enable* or *Mandatory*, establish how the username within each client certificate is to be determined.
 +
 Each certificate is expected to provide a _username_ as part of its content.
-Frequently, the username is specified as the certificate's _Subject Common Name_; but it may also be expressed as a _Subject Alternative Name_.
+This should be expressed as a _Subject Alternative Name_.
```

Comment:
I would not change this piece. As we are talking about _client_ certs here it's pretty accurate I beleive.

Reply:
Ok, I've restored the original.